client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```diff
@@ -81,13 +83,6 @@ $width-left-panel-card: calc( 55% );
 				}
 			}
 		}
-
-		@include breakpoint-deprecated( '<1040px' ) {
-			&,
-			&.no-gsuite {
-				width: 100%;
-			}
-		}
 	}
 }
 
```

Comment:
The `.action-panel__figure`, `.is-expanded .promo-card__title-badge` and `h2.action-panel__title` selectors on the following lines are too general -- please push them under a wrapping selector for this specific page.